src/Dockerfile

```diff
@@ -38,19 +38,26 @@ RUN <<EOT
 set -ex
 if ! yum install -y epel-release; then
   if . /etc/os-release 2>/dev/null; then
-    if [ "$ID" != "fedora" ]; then
-      yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${VERSION:0:1}.noarch.rpm
-    fi
+    case "$ID" in
+      fedora) ;;
+      rocky)
+        dnf install -y epel-release
+        ;;
+      *)
+        yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-${VERSION:0:1}.noarch.rpm
+        yum update -y
+        ;;
+    esac
   fi
 fi
+if command -v dnf &> /dev/null; then
```

Comment:
nit: I believe `&>` is bash-ism, although in these distros, probably sh is always actually same as bash.